pandas/core/groupby/generic.py

```diff
@@ -1691,17 +1697,17 @@ def _wrap_transformed_output(
 
         return result
 
-    def _wrap_agged_blocks(self, items, blocks):
+    def _agg_blocks_to_frame(self, items: Index, blocks: "List[Block]") -> DataFrame:
```

Comment:
I would actually prefer to keep this named `_wrap_agged_blocks` since it offers the same functionality as the rest of the `_wrap_*` functions in groupby, albeit with different input types

Reply:
That's a fair point, I wasn't aware of this systematic naming, I'll change it back. With the added typing we now see better that a frame is returned